pkg/kubelet/dockershim/docker_container.go

```diff
@@ -414,14 +414,10 @@ func (ds *dockerService) ContainerStatus(_ context.Context, req *runtimeapi.Cont
 	}
 
 	labels, annotations := extractLabels(r.Config.Labels)
-	imageName := r.Config.Image
-	if len(ir.RepoTags) > 0 {
-		imageName = ir.RepoTags[0]
-	}
 	status := &runtimeapi.ContainerStatus{
 		Id:          r.ID,
 		Metadata:    metadata,
-		Image:       &runtimeapi.ImageSpec{Image: imageName},
+		Image:       &runtimeapi.ImageSpec{Image: r.Config.Image},
```

Comment:
could we check whether r.Config.Image is an image ID here? Since the imageID has already been in ImageRef, I think it's better to show repotags in image if there're.

Reply:
repo tags usually show more meaningful information than imageID, hence I think we should still keep repo tags whenever possible. Since repo tags are not unique, I think it's OK to show different tags in different containers.